rootfs/usr/share/inputplumber/schema/capability_map_v2.json

```diff
@@ -0,0 +1,1257 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "CapabilityMapConfigV2",
+  "description": "[CapabilityMapConfigV2] are used to fix or define the real capabilities of an input device.",
+  "type": "object",
+  "required": [
+    "id",
+    "kind",
+    "mapping",
+    "name",
+    "version"
+  ],
+  "properties": {
+    "id": {
+      "type": "string"
+    },
+    "kind": {
+      "type": "string"
+    },
+    "mapping": {
+      "type": "array",
+      "items": {
+        "$ref": "#/definitions/CapabilityMapping"
+      }
+    },
+    "name": {
+      "type": "string"
+    },
+    "version": {
+      "type": "integer",
+      "format": "uint32",
+      "minimum": 0.0
+    }
+  },
+  "definitions": {
+    "AxisCapability": {
+      "type": "object",
+      "required": [
+        "name"
+      ],
+      "properties": {
+        "deadzone": {
+          "type": [
+            "number",
+            "null"
+          ],
+          "format": "double"
+        },
+        "direction": {
+          "type": [
+            "string",
+            "null"
+          ]
+        },
+        "name": {
+          "type": "string"
+        }
+      }
+    },
+    "AxisDirection": {
+      "description": "The AxisDirection is used to determine if a button value should be mapped towards the maximum axis value or the minimum axis value. For example, when mapping a BTN_UP to an ABS_HAT0Y, the converted value should be positive, towards that axis's maximum value, whereas BTN_DOWN should be Negative, towards that axis's minimum value.",
```

Comment:
Does this also account for exclusive pos/neg assignments? I.e. an event mapped 255 -> 0 as an up, will that trigger an event mapped 0 -> -254 as a down? Verbage might need some tweaking to avoid confusion.